src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager.php

```diff
@@ -160,7 +159,6 @@ private function decideConsensus(TokenInterface $token, array $attributes, $obje
                     break;
 
                 default:
```

Comment:
BTW @keradus, adding a new fixer dropping empty default blocks in switch-case would probably make sense for symfony preset.